### PR TITLE
Create test.js.yml

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: .github/workflows/test.js.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js 14.15.0
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.15.0'
+    - name: install browser dependencies
+      if: matrix.os == 'ubuntu-latest'
```
Comment (ramirobg94) [inline .github/workflows/test.js.yml:21]:
```suggestion
```

File: .github/workflows/test.js.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    - run: ln -s dist/server.js .
+    - name: run server
+      run: exec pm2-runtime server.js &
+      env:
+        PGHOST: localhost
+        PGPORT: 5444
+        PGDATABASE: of-arena-test
+        PGUSER: arena
+        PGPASSWORD: arena
+        SESSION_ID_COOKIE_SECRET: my-cookie-secret-key
```
Comment (sorja) [inline .github/workflows/test.js.yml:72]: Should we use github variables?

Comment (ramirobg94) [reply .github/workflows/test.js.yml:72]: yep I thinkt that should be a better way to set env vars

Comment (sorja) [reply .github/workflows/test.js.yml:72]: You can use .env file or use github secrets/env vars

**Pull request closed without merging**